docs/user/neuralnet.rst

```diff
@@ -326,20 +327,20 @@ predict(X) and predict_proba(X)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 These methods perform an inference step on the input data and return
-``numpy array``\s. By default,
+:class:`numpy.ndarray`\s. By default,
 :func:`~skorch.net.NeuralNet.predict_proba` will return whatever it is
 that the ``module``\'s :func:`~torch.nn.Module.forward` method
-returns, cast to a ``numpy array``. If
+returns, cast to a :class:`numpy.ndarray`. If
 :func:`~torch.nn.Module.forward` returns multiple outputs as a tuple,
 only the first output is used, the rest is discarded.
 
-If casting the :func:`~torch.nn.Module.forward`\-output to ``numpy``
-is impossible, you will get an error. In that case, you should
-consider returning a PyTorch :class:`~torch.Tensor` from your
-:func:`~torch.nn.Module.forward` method, as this tensor can be
-converted to a ``numpy`` array. Alternatively, consider using the
-:func:`~skorch.net.NeuralNet.forward_iter` method to generate outputs
-from the ``module``, or directly call ``net.module_(X)``.
+If the :func:`~torch.nn.Module.forward`\-output can not be meaningfuly
```

Comment:
Sorry, I noticed the typo in "meaningfuly" just now. But I wonder whether this word is needed here at all. If the passage just reads "... cannot be cast to ...", the meaning should be clear.

Reply:
Agree, fixed.